packages/amplify_datastore/example/ios/Podfile

```diff
@@ -32,9 +32,9 @@ target 'Runner' do
   use_modular_headers!
 
   # Point pods to internal source files
-  pod 'Amplify', path: '../../ios/internal/'
-  pod 'AWSPluginsCore', path: '../../ios/internal/'
-  pod 'AWSDataStorePlugin', path: '../../ios/internal/'
+  pod 'Amplify', path: '../../ios/internal/', :inhibit_warnings => true
```

Comment:
Q: What is the purpose of this change?

Reply:
CI logs were showing tons of lint warnings for these files, but since we they are from Amplify Swift we don't care. We're not going to modify these files. Adding this line prevents warning messages in CI, making those logs more readable.